Lib/inspect.py

```diff
@@ -140,7 +140,7 @@
 
 
 import abc
-from annotationlib import get_annotations  # re-exported
+from annotationlib import Format, get_annotations  # re-exported
```

Comment:
is this comment correct for `Format`? `"Format"` doesn't seem to be included in `__all__`

Reply:
```suggestion
from annotationlib import Format
from annotationlib import get_annotations  # re-exported
```